02_bubble/vinilapeddireddy/bubblesort_test.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+//To test the bubbleFunction with positive numbers
+func TestBubble_PositiveNumbers(t *testing.T) {
+	// Given
```

Comment:
If you're going to use the `Given`, `When` and `Then` comments, you should use them all. This applies to all tests in this file (except `TestMainOutput` which has all three).

I don't feel that the comments are needed all the time, but they are useful when learning as repetition of the structure of the test. If a test is small enough like these are, the comments sometimes just make it harder to read. I'll leave it up to you which way to go.